.github/update.py

```diff
@@ -11,65 +11,34 @@
-# Generate a translations.pri containing all languages in this repo
-
-with open(os.path.join(VPN_PROJECT_DIR, 'translations', 'translations.pri'), 'w') as qtTranslationProject:
-    qtTranslationProject.write('TRANSLATIONS += \\ \n')
-    for folder in os.listdir(OUT_PROJECT_DIR):
-        if not os.path.isdir(os.path.join(OUT_PROJECT_DIR, folder)):
-            continue
-        if folder in IGNORED_FOLDERS:
-            continue
-        qtTranslationProject.write(f'../translations/mozillavpn_{folder}.ts  \\ \n')
-    qtTranslationProject.write('\n \n ##End')
-
 srcFile = os.path.join(VPN_PROJECT_DIR, 'src', 'src.pro')
 os.system(f'lupdate {srcFile} -ts')
 
-for fileName in os.listdir(os.path.join(VPN_PROJECT_DIR, 'translations')):
-    if (not fileName.endswith('.ts')):
-        continue
-    filePath = os.path.join(VPN_PROJECT_DIR, 'translations', fileName)
-    # Usual filename: mozillavpn_zh-cn.ts
-    locale = fileName.split('_')[1].split('.')[0]  # de, zh-cn, etc.
-    baseName = fileName.split('_')[0]  # mozillavpn
-    outPath = os.path.join(OUT_PROJECT_DIR, locale)
-    # Create folder for each locale and convert
-    # ts file to /{locale}/mozillavpn.xliff
-    if not os.path.exists(outPath):
-        os.mkdir(outPath)
-
-    outFile = os.path.join(outPath, f'{baseName}.xliff')
-    if not os.path.exists(outFile):
-        # If the file doesn't exist
-        print(f'Creating {outFile}')
-        os.system(f'lconvert -i {filePath} -of xlf -o {outFile}')
-    else:
-        # Keep current translations
-        print(f'Updating {outFile}')
-        os.system(f'lconvert -if ts -i {filePath} -if xlf -i {outFile} -of xlf -o {outFile}')
-
-    # Now clean up the new xliff file
-    tree = ET.parse(outFile)
-    root = tree.getroot()
-
-    # Iterate all targetElements and remove empty ones
-    for element in root.iter('{urn:oasis:names:tc:xliff:document:1.2}trans-unit'):
-        target = element.find('{urn:oasis:names:tc:xliff:document:1.2}target')
-        if (not target.text):
-            element.remove(target)
-
-#    # Unescape any html in text
-#    for element in root.iter('{urn:oasis:names:tc:xliff:document:1.2}source'):
-#        t = element.text
-#        element.clear()
-#        element.tail = su.unescape(t)
-#        #print(f'Unescaped : {element.text}')
-
-    # Iterate all targetElements and remove empty ones
-    for element in root.iter('{urn:oasis:names:tc:xliff:document:1.2}extracomment'):
-        element.tag = 'note'
-    tree.write(outFile)
+fileName = "mozillavpn_en.ts"
+filePath = os.path.join(VPN_PROJECT_DIR, 'translations', fileName)
+# Usual filename: mozillavpn_zh-cn.ts
+locale = fileName.split('_')[1].split('.')[0]  # de, zh-cn, etc.
+baseName = fileName.split('_')[0]  # mozillavpn
+outPath = os.path.join(OUT_PROJECT_DIR, locale)
+outFile = os.path.join(outPath, f'{baseName}.xliff')
+# Create folder for each locale and convert
+# ts file to /{locale}/mozillavpn.xliff
+
+# Keep current translations
```

Comment:
--> Update English XLIFF file

Reply:
This one isn't fixed (we don't deal with translations anymore, we just export the English file)